equinox-shipping/Domain/FinalizationTransaction.fs

```diff
@@ -0,0 +1,91 @@
+module FinalizationTransaction
+
+module Events =
+
+    let [<Literal>] CategoryId = "FinalizationTransaction"
+    let streamName clientId = FsCodec.StreamName.create CategoryId clientId
```

Comment:
clientId -> transactionId (wrap as UMX)
CategoryId -> Category

Reply:
Updated and added UMX thanks!